From 147b93a029b5d57d21f2e2a04704ef498a9dd906 Mon Sep 17 00:00:00 2001
From: Masatake Iwasaki <iwasakims@apache.org>
Date: Tue, 5 Apr 2016 03:19:48 +0900
Subject: [PATCH 2360/2848] HDFS-9599.
 TestDecommissioningStatus.testDecommissionStatus
 occasionally fails (Lin Yiqun via iwasakims)

(cherry picked from commit 154d2532cf015e9ab9141864bd3ab0d6100ef597)
(cherry picked from commit c981efb0f67a4a2599689481e0efa8ae3a11d895)

Change-Id: I53b2e3d95297dec1ad6a189c9179d1a44ead6fc2
---
 .../server/namenode/TestDecommissioningStatus.java |   12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/server/namenode/TestDecommissioningStatus.java b/hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/server/namenode/TestDecommissioningStatus.java
index 8bf0bc7..f193ba4 100644
--- a/hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/server/namenode/TestDecommissioningStatus.java
+++ b/hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/server/namenode/TestDecommissioningStatus.java
@@ -58,8 +58,8 @@
 import org.apache.hadoop.hdfs.tools.DFSAdmin;
 import org.apache.log4j.Level;
 import org.apache.log4j.Logger;
-import org.junit.AfterClass;
-import org.junit.BeforeClass;
+import org.junit.After;
+import org.junit.Before;
 import org.junit.Test;
 
 /**
@@ -79,8 +79,8 @@
 
   final ArrayList<String> decommissionedNodes = new ArrayList<String>(numDatanodes);
   
-  @BeforeClass
-  public static void setUp() throws Exception {
+  @Before
+  public void setUp() throws Exception {
     conf = new HdfsConfiguration();
     conf.setBoolean(DFSConfigKeys.DFS_NAMENODE_REPLICATION_CONSIDERLOAD_KEY,
         false);
@@ -114,8 +114,8 @@ public static void setUp() throws Exception {
     Logger.getLogger(DecommissionManager.class).setLevel(Level.DEBUG);
   }
 
-  @AfterClass
-  public static void tearDown() throws Exception {
+  @After
+  public void tearDown() throws Exception {
     if (localFileSys != null ) cleanupFile(localFileSys, dir);
     if(fileSys != null) fileSys.close();
     if(cluster != null) cluster.shutdown();
-- 
1.7.9.5

